src/Honeybee_Set EnergyPlus Zone Loads.py

```diff
@@ -114,7 +114,10 @@ def main(HBZones, equipmentLoadPerArea, infiltrationRatePerArea, lightingDensity
         
         if equipmentLoadPerArea[zoneCount] != None:
             HBZone.equipmentLoadPerArea = equipmentLoadPerArea[zoneCount]
-        if infiltrationRatePerArea[zoneCount] != None:
+        if infiltrationRatePerArea[zoneCount] == None:
+            w = gh.GH_RuntimeMessageLevel.Warning
+            ghenv.Component.AddRuntimeMessage(w, "No infiltration rate per floor area has been set, please use Honeybee_infOrVentPerArea to convert infiltration rate per Facade area as an input!")
```

Comment:
This warning here doesn't seem appropriate since it's odd to give a warning about infiltration and nothing else.

I just checked the `OpenStudio_Standards.json` (https://github.com/ladybug-tools/honeybee-legacy/blob/master/resources/OpenStudio_Standards.json) and realized that you were right that the infiltration values are supposed to be per m2 facade and not m2 floor. It looks like @mostaphaRoudsari just missed this when he was first setting up the honeybee plugin and I didn't know where to check his assumption at the time so I blindly took it as fact that they were supposed to be per m2 of floor. 

In light of this, what if we instead just edited the OpenStudio component and IDF component to use infiltration per exterior area? This should just involve the change of a single line of code in each component and so I think we can do it as part of this PR. I also realize that this will make the `Honeybee_infOrVentPerArea` obsolete so maybe we should just move it into WIP for now.

This way, I will correct any hydra examples using the old infiltration method for the next (final) release of legacy that should be pretty soon. What do you think about this @MingboPeng ?

Reply:
Hi @chriswmackey, I am not sure if it is appropriate to change it to per m2 facade in current workflow:
1. all other inputs in "SetZoneLoads", like numOfPeople, Equp, LPD, etc, are all W/floor m2. It might be even misleading for users who doesn't check each input carefully.
2. changing units from per m2 floor to m2 facade, will break users' current workflow, and leads to mismatching results with the "same" gh file (before and after update). Most of users probably won't even notice that we have changed the unit for infiltration.
3. instead of making `Honeybee_infOrVentPerArea` obsolete, we can add another component, something like this. (not sure if we really want to do this, I can imagine that there would be a lot code changes in "ExportToOpenStudio" component )
![image](https://user-images.githubusercontent.com/9031066/67427932-1ca07000-f5ab-11e9-83da-13abf940d5d2.png)
4. Add a warning here is the smallest place that I can find that has minimum impact to both legacy code and user's workflow, and also raise an attention that there is something wrong about the infiltration.

In terms of hydra examples, we can add Honeybee_infOrVentPerArea to calculate infiltration rate per floor area just for final release. please let me know. 
![image](https://user-images.githubusercontent.com/9031066/67429582-78202d00-f5ae-11e9-9a18-296bc88cdd02.png)

Reply:
I can say that I am positive that the values we pull from the OpenStudio_Standards.json are supposed to be per unit area of exposed envelope (I don't know what else the `infiltration_per_area_ext` key in the json would be referring to). 

And ok but adding a warning message that displays every time someone doesn't set the infiltration seems almost just as disruptive to everyone's current workflows as just fixing the default assumption. Especially given that the warning message doesn't explain why users are supposed to manually change the default infiltration and not any other attributes. And, if we're going to write a warning message that says "the default values for infiltration within this plugin are incorrect", then maybe we should just fix it. If we fix it to use the correct method, then we could fix the Honeybee_infOrVentPerArea component to use the zone exposed envelope area instead of floor area and add a warning to let people know that the calculation the component does has changed. BTW this change should only affect the users of the ACH method and does not affect the flow/m2 facade method (so we really only need to throw a warning for that case).

If you think it's a lot of things to fix, I am happy to just send a PR myself that addresses it. This bug is largely my fault after all and I should have known to check the json file sooner. Just let me know what you prefer. If you'd rather I make the edits, we can just merge this PR in and I'll send another PR, which you can look over to be sure it's not disrupting your current workflows.